src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -3,26 +3,26 @@
   <div class="row mx-0">
     <cd-card title="Details"
              i18n-title
-             class="col-sm-3 px-3">
+             class="col-sm-3 px-3 d-flex">
       <dl class="ms-4 me-4">
         <dt>FSID</dt>
         <dd>{{ detailsCardData.fsid }}</dd>
         <dt>Orchestror</dt>
-        <dd>{{ detailsCardData.orchestrator }}</dd>
+        <dd i18n>{{ detailsCardData.orchestrator || 'N/A' }}</dd>
```

Comment:
Instead of `N/A` what about a meaningful message as "Orchestrator module disabled"

Reply:
Fixed the test issue,


> Instead of `N/A` what about a meaningful message as "Orchestrator module disabled"

I went with `N/A` as I though it is the default value we were using in the Dashboard to display when no value was to be displayed (Not Applicable)

Reply:
I agree with @pereman2. Showing an N/A would be more confusing than helpful in this situation. I know its used in the host list more often but it is usually followed up with an hovering texts saying that `Orchestrator is disabled` or a similar one. If we are displaying N/A, then we should be clear why its showing an N/A. So either an info icon right next to which holds the `Orchestrator is disabled` message or just a plane text with `Orchestrator is disabled`.

Or an alternate would be to completely disable the `Orchestrator` item from the Details view when it is disabled. Its a new thing and no one will miss it I guess. That way we can keep the Details view populated with informations that are available.

Reply:
In the end I went with: `Orchestrator is not available` to keep the consistency as is the same message we are displaying in other parts of the dashboard when the orchestror is disabled.